app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -443,13 +444,17 @@ fun PocketStoriesCategories(
             verticalItemsSpacing = 16.dp,
         ) {
             categories.filter { it.name != POCKET_STORIES_DEFAULT_CATEGORY_NAME }.forEach { category ->
+                val isSelected = selections.map { it.name }.contains(category.name)
+
                 SelectableChip(
                     text = category.name,
-                    isSelected = selections.map { it.name }.contains(category.name),
-                    selectedTextColor = categoryColors.selectedTextColor,
-                    unselectedTextColor = categoryColors.unselectedTextColor,
-                    selectedBackgroundColor = categoryColors.selectedBackgroundColor,
-                    unselectedBackgroundColor = categoryColors.unselectedBackgroundColor,
+                    isSelected = isSelected,
+                    colors = SelectableChipColors(
+                        selectedTextColor = categoryColors.selectedTextColor,
+                        unselectedTextColor =  categoryColors.unselectedTextColor,
+                        selectedBackgroundColor = categoryColors.selectedBackgroundColor,
+                        unselectedBackgroundColor = categoryColors.unselectedBackgroundColor,
+                    )
```

Comment:
It might make more sense to break this instantiation outside of the loop and maybe above `Box`. Let's also use `SelectableChipColors.buildColors` in case  more colors are ever added to `SelectableChipColors` and they don't need to be forced downstream here.